cdap-docs/faqs/source/cdap.rst

```diff
@@ -87,6 +64,20 @@ If you have followed :ref:`the installation instructions <install>`, and CDAP ei
   installed the packages and started the services).
 
 
+Using YARN with the LinuxContainerExecutor.
```

Comment:
While there is clearly no ordering of the questions, this one really sticks out since the questions above and below are closely related.

suggested order:

(config/general)
What are the memory and core requirements for CDAP?
How do I set the CDAP properties for components running on multiple machines?
Using YARN with the LinuxContainerExecutor.
Where is the CDAP CLI (Command Line Interface) in Distributed mode?


(install/yarn-related)
I’ve followed the install instructions, yet CDAP does not start and fails verification. What next?
The CDAP UI is showing a message “namespace cannot be found”.
I don’t see the CDAP Master service on YARN.
YARN Application shows ACCEPTED for some time but then fails.

(install/general)
The CDAP Master log shows permissions issues.
The CDAP Master log shows an error about the dataset service not being found.
The HiveServer2 already listens on port 10000; what should I do?
CDAP services on distributed CDAP aren’t starting up due to java.lang.ClassNotFoundException. What should I do?
We aren’t seeing any Metrics or Logs. What should we do?
Only the first flowlet of our CDAP application is showing activity.
Log Saver Process throws an Out-of-Memory Error; the CDAP UI shows service “Not OK”

(upgrade)
Can a CDAP installation be upgraded more than one version?